cartographer/field_types/schema_relationship.py

```diff
@@ -56,7 +56,7 @@ def resource(self, parent_resource, relationship_key):
         if self.id_attribute is not None:
             relationship_model_class = self.resource_registry_entry().get(ResourceRegistryKeys.MODEL)
             model_id = getattr(parent_resource.model, self.id_attribute)
-            if model_id is not None:
+            if model_id is not None and hasattr(relationship_model_class, 'get'):
```

Comment:
Correct me if this already exists, but what do you think about having documentation for what a `model` object is? It seems to implicitly be a sqlalchemy model, but I know that's not a requirement. It'd be good to know what interface we expect models to have and enforce it here.

This is one of those times where I wish we had better debug capabilities for Cartographer, because a consumer of this library might not know that `get()` is necessary and then wonder what's going on when this check fails silently.